### ISSUE TITLE
2024_feb_roadmap #story_l: Test Self-Registration

### ISSUE THREAD

**Issue by ljmoody** [lzim/teampsd#3026]
Laura and Jenn test the facility self-registration at https://mtl.how/test.

**To test 2/15:**
- [x] test point 15
- [ ] test point 17 - _dev is working on proof of concept_

**Comment by lijenn** [comment]
@tejaspesquare Please see @ljmoody  and my user test feedback below:
- [ ] 1. The user's selected VISN, Facility, and Clinic is missing in the team name pop-up:
![image](https://github.com/lzim/teampsd/assets/59668647/9656f80e-f92c-4d04-82b0-f9ffaf35ed95)
- For example, if the user selected VISN 19, Facility 554, and Clinic GJ, the highlighted yellow section should pre-populate: v19_554_gj_
- The team name should always contain the VISN, Facility, and Clinic, in addition to the custom name the user inputs.

- [ ] 2. Delete the "at" in the Approval email:
![image](https://github.com/lzim/teampsd/assets/59668647/08280bdf-3f65-4b88-b7ce-042b3b571e9f)

- [ ] 3. The "Approve" button should be green.
![image](https://github.com/lzim/teampsd/assets/59668647/77fce4fc-7fac-462f-8847-8701f5b38d2b)

- [ ] 4. The team name's VISN, Facility, and Clinic are missing.
![image](https://github.com/lzim/teampsd/assets/59668647/60649f60-6f82-4887-bfe5-b89432602821)

- [ ] 5. In Shared worlds, change "Team" in the upper left hand corner to "Shared" so users know they are in the Shared world and not the Team world.
![image](https://github.com/lzim/teampsd/assets/59668647/677a3c18-8219-493e-b9cd-e96772ae0ca7)

- [ ] 6. All teams that register in the self-registration must show up users who have the administrative_login privilege. 
For example, Laura registered the team. But when Jenn logs in with her administrative_login, Jenn does not see the team:
![image](https://github.com/lzim/teampsd/assets/59668647/c0d024b0-3948-42ac-a207-ca9915861e62)
![image](https://github.com/lzim/teampsd/assets/59668647/503d9459-002d-4584-b9f4-5c905f6e1e31)

Let us know if you have any questions!

**Comment by ljmoody** [comment]
Hi @tejaspesquare - thanks for giving a 👍 so we know you see messages @lijenn and I post. We would like to test the self-registration process and documentation with users next week (on Nov 29). 
- [ ] Are you able to give us an estimated date of completion for the feedback items above?

**Comment by lijenn** [comment]
@tejaspesquare Users who have the administrative_login privilege should have access to any team that is set up or exists in the Sim UI (whether the user was added to the team or not). 

For example, if a doctor created a team/world using the self-registration, anyone who has the administrative_login privilege should have access to the team that the doctor created. Assume that they did not add Laura and I to the team/world either. 

However, because Laura and I have the administrative_login privilege, Laura and I could still access and see the doctor's team/world without being added as a member. This is in case the doctor has questions in the world/team, Laura and I can access it to help them.

This is the administrative_login:
![image](https://github.com/lzim/teampsd/assets/59668647/d68ec727-5321-4af1-a150-e1b28c27a09a)

Please let me know if you need more clarification and if we need to meet to talk about this!

**Comment by tejaspesquare** [comment]
@lijenn   Thank you for detailed response. I think it answered now. Whoever is logged in with administrator_login world will be able to see entire list of teams available in SIM UI.

Here self registration follows for this step, correct?  Or do we have a separate self registration portal exist & I'm not aware?
i.e. Login -> Register for First Time

![Self Registration](https://github.com/lzim/teampsd/assets/144898781/8f218731-4fa2-4683-a908-96f15133a972)

**Comment by lijenn** [comment]
@tejaspesquare Yes! That screen is the self-registration process.

**Comment by lijenn** [comment]
@tejaspesquare Laura and I got to test with members on our team, here's more feedback.

Note for point 6: I believe the feature was working before Hiren left. So there may just be a bug somewhere/unsure why the feature stopped working.

- [ ] 7. Add a blank/empty selection as the default selection for each VISN's Clinic field. 
- The Clinic field is optional so users do not have to select a Clinic. For example, if a user selects VISN 21 and Facility 640, the Clinic field should have a blank/empty field as the default. 
Note: If a user does not select a Clinic, their team name would exclude the Clinic. 
Example: v21_640_jennifers_team_name
![image](https://github.com/lzim/teampsd/assets/59668647/038ff165-5d3a-42ac-8c48-28099b6ae3cb)

- [ ] 8. There should be no underscore between the Facility and Clinic in the team name. Append the selected Clinic directly after Facility. So this pop-up below should be: v21_640hc
![image](https://github.com/lzim/teampsd/assets/59668647/ccc0a008-6878-44a9-822f-173f7e9aba7f)
![image](https://github.com/lzim/teampsd/assets/59668647/0926107e-f01b-442b-9d52-241e84124b6b)

- [ ] 8. Add the subtext "Must contain at least 1 number" underneath "New Password" and "Repeat Password". Use the same font and font size as the "Sta3n" and "Sta6a" text.
![image](https://github.com/lzim/teampsd/assets/59668647/91aa39cd-971c-4acd-bf4e-feeb088e918e)

- [ ] 9. Change wording from "Select Facility Team" to "Select Or Create A Facility Team".
![image](https://github.com/lzim/teampsd/assets/59668647/ea463b0f-2130-411f-b909-4443e1402240)

- [ ] 10. Change wording from "Enter your facility team name" to "Complete Your Facility's Team Name".
![image](https://github.com/lzim/teampsd/assets/59668647/246f75ee-ce0f-48e8-9b8f-e55e8c4ce814)

- [ ] 11. Add the subtext "Our team naming convention uses all lower-case and underscores. For example: v21_640a0_ mpd_green_bhip" underneath "Enter your facility team name". Use the same font and font size as the "Sta3n" and "Sta6a" text.
![image](https://github.com/lzim/teampsd/assets/59668647/ea41846a-f0c3-4d4f-8c64-8327fbea92f7)

- [ ] 12. Missing underscore after the Clinic in the team name pop-up
![image](https://github.com/lzim/teampsd/assets/59668647/a4019040-3fd3-44c0-9c12-12004c50c647)

- [ ] 13. Previously created/existing teams are not showing up. I created 1 team using the same VISN, Facility, and Clinic and could not find it in the dropdown:
![image](https://github.com/lzim/teampsd/assets/59668647/24d0f17b-3278-4bf2-a576-5da7bc32eade)

- [ ] 14. Change "Your registration has been forwarded to the Approver." to "Your registration has been forwarded to _MTL_ for approval." Use the same font and font size as the "Sta3n" and "Sta6a" text.
![image](https://github.com/lzim/teampsd/assets/59668647/9a33417e-2202-4b3d-87d7-53e9b4be6539)

- [ ] 15. Hide the Shared world for all users that do not have the administrative_login privilege. Only users who have the administrative_login should be able to see all the registered teams, ind, and shared worlds.
![image](https://github.com/lzim/teampsd/assets/59668647/cf9df40a-dfc5-456d-a201-4365cf5c040c)

---------------------------------------------------
These 2 errors occurred in the team, shared, and individual worlds for users who registered using the self-registration form .

- [ ] 16. The Delete and Upload button do not work.
![image](https://github.com/lzim/teampsd/assets/59668647/990d08af-eb38-4a35-8193-536433b2fee1)

- [ ] 17. Users who registered using the self-registration form should **not** be able to see all the existing data files in the Sim UI. We do not want users to be able to access and view every single data file from other people and clinical teams uploaded in the Sim UI.
![image](https://github.com/lzim/teampsd/assets/59668647/6841f715-5c36-446a-a0f6-0141d5a8955c)

They should just see the data files they:
- A) choose to Upload (using the Upload button) and 
![image](https://github.com/lzim/teampsd/assets/59668647/16540331-3c8f-497b-b8c9-de78b2792ab3)

- B) the list of data files that an administrator_login user adds into their world
![image](https://github.com/lzim/teampsd/assets/59668647/78f62bc3-abdf-4298-a395-6608049fa2d7)

**Comment by tejaspesquare** [comment]
@lijenn 
1.Point 7 & 8 - We have added additional drop-down value under clinic as 'select' so that it will work according to notes for Point 7 & 8.  Otherwise, it will have one clinic value default selected.

2.Point 15 - It required to analyze existing code which allow us to filter shared worlds who do not have administrator_login  privilege.

3.Point 16 - Delete & upload button works for us.
![image](https://github.com/lzim/teampsd/assets/144898781/1b5a750a-1b60-4f10-8662-991c7306dc07)

After Delete
![image](https://github.com/lzim/teampsd/assets/144898781/3d27e96b-3f7e-4688-ac07-4074e1573c18)

4.Point 17 - It required to investigate functionality & possibility to filter records based on self registered users. Need to find API support this.

**Comment by tejaspesquare** [comment]
@lijenn Development on points 7 To 14 is completed & available on dev environment. I will check with James if we can proceed to deploy on TEST-Slow.

**Comment by tejaspesquare** [comment]
@ljmoody Changes are moved to test. Kindly verify & let us know your comments.   Note:  Please ignore Flow Meter of 'Team Flow'. Kept on hold & suggest if required to hide it.
Sincerely apologize for any inconvenience.

**Comment by ljmoody** [comment]
Hi @tejaspesquare. Please incorporate the comments below on self-registration windows and popups:

- [ ] 1. On the self-registration form, please uncapitalize the O and A (highlighted in yellow); I missed that detail before:
<img width="400" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOWMwQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--9cbbb05e20231308d855fa3b1552ba0d0127f07e/image.png" alt="image.png" />

- [ ] 2. On the first person to register popup:
- Change "lower-case" (highlighted in yellow) to "lowercase"
- Please remove the space (highlighted in yellow) in the example given so it reads: "v21_640a0_mpd_green_bhip"
- There should be no underscore between the Facility and Clinic in the team name (highlighted in yellow). Append the selected Clinic directly after Facility. 
- There is a missing underscore after the Clinic in the team name pop-up. So, this pop-up should read: v19_554bu_
-The "u" in "bu" looks cut off. Please check that this entire field's pre-populated letters and characters are fully seen.
<img width="436" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOEEwQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f04a3ae9de91e990bf8bb1b8dca2c9b691dac875/image.png" alt="image.png" />

- [ ] 3. The popup approver message I'm seeing is still the same. From point 14 above, change "Your registration has been forwarded to the Approver." to "Your registration has been forwarded to _MTL_ for approval." Use the same font and font size as the "Sta3n" and "Sta6a" text.
<img width="466" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOVEwQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--622d09765c9931b8f0491e5d494a36f42a3dd571/image.png" alt="image.png" />

- [ ] 4. In the approval email, please (highlighted in yellow):
- change T to lowercase
- change the comma to a period:
<img width="366" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL1UwQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a30eed98b57f3a651a8f6f0bfc94252a3f6774c9/image.png" alt="image.png" />

@lijenn - FYI

**Comment by tejaspesquare** [comment]
> Hi @tejaspesquare. Please incorporate the comments below on self-registration windows and popups:
> 
> * [ ]  1. On the self-registration form, please uncapitalize the O and A (highlighted in yellow); I missed that detail before:
> 
> <img alt="image.png" width="400" src="https://camo.githubusercontent.com/2f836e2853922cd1fb3ccded69282c6fd60349e93ac8cfc9527187edcabde1e1/68747470733a2f2f6170692e7a656e6875622e636f6d2f617474616368656446696c65732f65794a66636d467062484d694f6e73696257567a6332466e5a534936496b4a42614842424f574d775157633950534973496d563463434936626e56736243776963485679496a6f69596d7876596c39705a434a3966513d3d2d2d396362626230356532303233313330386438353566613362313535326261306430313237663037652f696d6167652e706e67">
> 
> * [ ]  2. On the first person to register popup:
> * Change "lower-case" (highlighted in yellow) to "lowercase"
> * Please remove the space (highlighted in yellow) in the example given so it reads: "v21_640a0_mpd_green_bhip"
> * There should be no underscore between the Facility and Clinic in the team name (highlighted in yellow). Append the selected Clinic directly after Facility.
> * There is a missing underscore after the Clinic in the team name pop-up. So, this pop-up should read: v19_554bu_
>   -The "u" in "bu" looks cut off. Please check that this entire field's pre-populated letters and characters are fully seen.
> 
> <img alt="image.png" width="436" src="https://camo.githubusercontent.com/8490f2818bee0f129a4dbb646121c364c9a27acdcc5c83c62ee21dcd24874d59/68747470733a2f2f6170692e7a656e6875622e636f6d2f617474616368656446696c65732f65794a66636d467062484d694f6e73696257567a6332466e5a534936496b4a42614842424f4545775157633950534973496d563463434936626e56736243776963485679496a6f69596d7876596c39705a434a3966513d3d2d2d663034613361653964653931653939306266386262316238646361326339623639316461633837352f696d6167652e706e67">
> 
> * [ ]  3. The popup approver message I'm seeing is still the same. From point 14 above, change "Your registration has been forwarded to the Approver." to "Your registration has been forwarded to _MTL_ for approval." Use the same font and font size as before, don't change the font/size to the "Sta3n" and "Sta6a" text.
> 
> <img alt="image.png" width="466" src="https://camo.githubusercontent.com/1cb37bd7a0e14b4dca5762d56da2c9f87543ede4c4c8930c5f6107d8b4965595/68747470733a2f2f6170692e7a656e6875622e636f6d2f617474616368656446696c65732f65794a66636d467062484d694f6e73696257567a6332466e5a534936496b4a42614842424f5645775157633950534973496d563463434936626e56736243776963485679496a6f69596d7876596c39705a434a3966513d3d2d2d363232643039373635633939333162386630343931653564343934613336663432613364643537312f696d6167652e706e67">
> 
> * [ ]  4. In the approval email, please (highlighted in yellow):
> * change T to lowercase
> * change the comma to a period:
> 
> <img alt="image.png" width="366" src="https://camo.githubusercontent.com/7e142a32db761c7e820ef2b4b7cc4a851fb9bef750423bdff621de81ae47391f/68747470733a2f2f6170692e7a656e6875622e636f6d2f617474616368656446696c65732f65794a66636d467062484d694f6e73696257567a6332466e5a534936496b4a42614842424c3155775157633950534973496d563463434936626e56736243776963485679496a6f69596d7876596c39705a434a3966513d3d2d2d613330656564393862353766336136353161386636663062666339343235326133663637373463392f696d6167652e706e67">
> 
> @lijenn - FYI

@ljmoody  - Can't access attachments. Could you please reshare it for our reference.

**Comment by tejaspesquare** [comment]
@ljmoody @lijenn - Point 3 (5th Dec communication) - Popup approver message working fine. Not sure why it is not reflected on your side. Do clear cache & delete histories may fix issue automatically.

![image (1)](https://github.com/lzim/teampsd/assets/144898781/367ec9d2-446c-4ff2-bf81-378b394a6222)

**Comment by tejaspesquare** [comment]
@ljmoody @jamesmrollins @lijenn - Regarding Point No 17.  'Users who registered using the self-registration form should not be able to see all the existing data files in the Sim UI. We do not want users to be able to access and view every single data file from other people and clinical teams uploaded in the Sim UI.'


Actually, we were looking for API which retrieve list of data files uploaded by self registered user & based on results we can differentiate file uploaded by self registered user or administrator_login user. Current API result information is not enough to  We contacted David for the same & received response as below:

"The project file system is, unsurprisingly, backed by a file system. The owner of that file system is necessarily the entity executing file operations, which is epicenter. Neither your users nor anyone else’s holds an actual user account as far as the file system is concerned, so we have no way to provide that information. However, if you’re making the api calls, I would guess you do, and could keep a history, or shadow file information. You would need a convenient data storage system for that kind of information. We do provide the vault api for just this sort of usage. It is a storage system for loosely structured data of your choosing (actually a json store, actually backed by mongodb). The vault api…

https://forio.com/api/v3/account/project/encyclopedia/as/asciidoc_to_html/v3/vault

…can be complex, but utilizes the same building blocks as the rest of the v3 api, such as scope and permit. Proper usage takes some thought and planning, as does the proper usage of any data store, towards capabilities and long term data evolution. "


So, at this moment we are unsure about efforts & possible solution however we can give a try when time permits.

**Comment by ljmoody** [comment]
Hi @tejaspesquare. I'm sorry about this:
> @ljmoody - Can't access attachments. Could you please reshare it for our reference.

I've revised the attachments and my comments in the post above. Clearing the cache did change some things; thank you for suggesting that and being patient. 

I can confirm that this message is corrected now: 
> @ljmoody @lijenn - Point 3 (5th Dec communication) - Popup approver message working fine. Not sure why it is not reflected on your side. Do clear cache & delete histories may fix issue automatically.

**Comment by tejaspesquare** [comment]
> Hi @tejaspesquare. Please incorporate the comments below on self-registration windows and popups:
> 
> * [x]  1. On the self-registration form, please uncapitalize the O and A (highlighted in yellow); I missed that detail before:
>   ![1](https://private-user-images.githubusercontent.com/124643236/288500915-2f08a4e0-9bdc-445c-8fbc-9980fe7631b8.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE5NTM0MjcsIm5iZiI6MTcwMTk1MzEyNywicGF0aCI6Ii8xMjQ2NDMyMzYvMjg4NTAwOTE1LTJmMDhhNGUwLTliZGMtNDQ1Yy04ZmJjLTk5ODBmZTc2MzFiOC5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwN1QxMjQ1MjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01YWZhM2M3Yjk2YjQwZmZhNGM5ZGNjYzQ2YTg3MWUyYWZhMjY0ZmE5OWI1YzQ3NjQ0ZDAzZDc3Mjg1Yzk3MTlmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.lNAPnDE212F-zrOA7Mk0VspZHhWRjfwVc8vEIzzy9sY)
> * [x]  2. On the first person to register popup:
> * Change "lower-case" (highlighted in yellow) to "lowercase"
> * Please remove the space (highlighted in yellow) in the example given so it reads: "v21_640a0_mpd_green_bhip"
>   ![2](https://private-user-images.githubusercontent.com/124643236/288501121-27d97b82-7d9a-4b32-b729-d87012268337.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE5NTM0MjcsIm5iZiI6MTcwMTk1MzEyNywicGF0aCI6Ii8xMjQ2NDMyMzYvMjg4NTAxMTIxLTI3ZDk3YjgyLTdkOWEtNGIzMi1iNzI5LWQ4NzAxMjI2ODMzNy5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwN1QxMjQ1MjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03NmUwMGNmYzcxYzQzNTM3MzAyYmMzNTg0MTRkOTE0NjllMmUwNWU0N2MwZjM4Y2MwNjY5MjNjOWMxZTA1ZjQ4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.BYPmZJ5gIiAwGHCcCJ5vEJZNENWNGuTwCLpBr5HWHqI)
> * [x]  3. The popup approver message I'm seeing is still the same. From point 14 above, change "Your registration has been forwarded to the Approver." to "Your registration has been forwarded to _MTL_ for approval." Use the same font and font size as before, don't change the font/size to the "Sta3n" and "Sta6a" text.
> * [x]  4. In the approval email, please (highlighted in yellow):
> * change T to lowercase
> * change the comma to a period:
>   ![4](https://private-user-images.githubusercontent.com/124643236/288501799-1ce72595-b100-4020-85de-fce1c5aae034.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE5NTM0MjcsIm5iZiI6MTcwMTk1MzEyNywicGF0aCI6Ii8xMjQ2NDMyMzYvMjg4NTAxNzk5LTFjZTcyNTk1LWIxMDAtNDAyMC04NWRlLWZjZTFjNWFhZTAzNC5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwN1QxMjQ1MjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zZmRhMTg5ZjFiYzc0YTRlNjM0MjY1ZDg1ZmI5ZWRlY2Y5NTNmMWIxNmNkOWY0OGFkZjI4Njg0NGZiNmQ4NzZiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.mKc1VxIXRZOFI6ge3RdxeBukNjE5kzfpB38W9CRRlWg)
> 
> @lijenn - FYI

@ljmoody - Above changes are moved to TEST-Slow environment.

**Comment by lijenn** [comment]
Thanks @tejaspesquare! Here's the latest feedback. We're almost there!
- [ ] 18. Change "Register for First Time" to "Create Account or Team"
<img width="372" alt="image" src="https://github.com/lzim/teampsd/assets/59668647/9657f624-0bd7-47d8-8bee-297eeced8ed4">

- [ ] 19. Change "Create an Account or Team"
<img width="444" alt="image" src="https://github.com/lzim/teampsd/assets/59668647/31e30556-4057-4bd5-a22d-6d5d3f1ff9f7">

- [ ] 20. When selecting a Clinic/Sta6a, the existing teams for that Clinic/Sta6a do not show up in the "Select or Create a Facility Team" dropdown:
<img width="423" alt="image" src="https://github.com/lzim/teampsd/assets/59668647/340f3c6a-d826-4325-baee-32a0a6a30cdc">

- [ ] 21. For the subtext underneath: Complete Your Facility's Team Name
- Move "For" to the next line so "For example: v21..." is all on the same line.
- Increase the font size by 1 or 1.5x
- Bold the words "lowercase", "underscores", and "For example: v21_640a0_mpd_green_bhip". 
- Remove the parentheses
<img width="485" alt="image" src="https://github.com/lzim/teampsd/assets/59668647/3eab7c1b-336d-4e55-b790-a4d2ded2eb5b">

- [ ] 22. In the approver page, rename "Return" to "Reject".
<img width="477" alt="image" src="https://github.com/lzim/teampsd/assets/59668647/5ec3e893-579e-4a0a-a36b-8412f55fb245">

- [ ] 23. In the Return/Reject email, please follow this template:

Dear Jennifer,

Sorry! We had trouble registering your account or team.

Please use our team naming convention for your team name:
- All lowercase
- Use underscores instead of spaces
- Example: v21_640a0_mpd_green_bhip

_Need more help?_ Email: MTL.Help@va.gov (note to dev: please hyperlink the MTL.Help@va.gov email)

Go back to the sign up page here. (note to dev: use the link currently in "Go Back" in the old email template and put it in "here" in the new template).

Thanks!
_Modeling to Learn_ (note to dev: please italicize "Modeling to Learn" and "Need more help?")

<img width="562" alt="image" src="https://github.com/lzim/teampsd/assets/59668647/37794518-168f-44b3-a07d-cda9e2519cb4">

- [ ] 24. Change the colon to a period in the approval email. Unbold the "Thanks".
<img width="351" alt="image" src="https://github.com/lzim/teampsd/assets/59668647/826e63e1-4e23-4926-809f-a506bba876e3">

- [ ] 25. Remove "Facilitator" in the successful registration email. Turn the comma into an exclamation mark. Unbold the "Thanks".
<img width="873" alt="image" src="https://github.com/lzim/teampsd/assets/59668647/d557dd75-064c-435b-90bd-b2b67038c5bf">

- [ ] 26. When the screen is loading after clicking the green "GO!" button, include quick tips while the page is loading:
<img width="1766" alt="image" src="https://github.com/lzim/teampsd/assets/59668647/2c7659b1-35a3-4f29-8c8d-76ed6147e200">

For example, here's a random quick tip after I click "Play":
<img width="1782" alt="image" src="https://github.com/lzim/teampsd/assets/59668647/88e81843-d8f6-4c41-a65f-00ad1cd8d0c7">

All quick tips are located here:
<img width="1295" alt="image" src="https://github.com/lzim/teampsd/assets/59668647/68750030-f18a-4a74-a9af-20abc1eb9c1c">

-----
FYI @ljmoody, recommended some new language above, so screenshots might need to change. Lmk if you need help with that!

**Comment by tejaspesquare** [comment]
@lijenn - From latest list change pending for Point no 26 (quick tips). Rest all points are moved to TEST-Slow. 
Point 20 - We have not made any change. Can you please check once if it is working or not

![image](https://github.com/lzim/teampsd/assets/144898781/f810fbbb-17b3-4b05-aae0-38ff56fd730a)

**Comment by tejaspesquare** [comment]
> Hello @tejaspesquare ,
> 
> 26. Please italicize MTL in the popup message below:
> 
> ![approve popup](https://private-user-images.githubusercontent.com/124643236/289158240-b1287d8f-32bd-4904-a38c-a2ea6f47dd62.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzMDMwNDMsIm5iZiI6MTcwMjMwMjc0MywicGF0aCI6Ii8xMjQ2NDMyMzYvMjg5MTU4MjQwLWIxMjg3ZDhmLTMyYmQtNDkwNC1hMzhjLWEyZWE2ZjQ3ZGQ2Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxMVQxMzUyMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yNjY0ZTlmNGE3YTZlNjUyODkzNTA0ZmE5NDliYmU3MDc2ZGQ3NGQ4ZmZjYjUyZmRhNDkyZTQ4MDVmZTFiNTc3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.wpsoD8PzU_zmPcfmftDzTp9eqZ-PB5T8-5WHxv32AK0)

@ljmoody - This is updated & can be verified in TEST-Slow.

**Comment by lijenn** [comment]
Hi @tejaspesquare Laura and I do not see the MTL being italicized in the popup yet. I checked in both TEST-Slow and Dev but I still see this:

<img width="541" alt="image" src="https://github.com/lzim/teampsd/assets/59668647/3272e662-5db5-4762-ab28-f477c89e1cf3">

**Comment by tejaspesquare** [comment]
> Hi @tejaspesquare Laura and I do not see the MTL being italicized in the popup yet. I checked in both TEST-Slow and Dev but I still see this:
> 
> <img alt="image" width="541" src="https://private-user-images.githubusercontent.com/59668647/289644208-3272e662-5db5-4762-ab28-f477c89e1cf3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzNTQ3NTEsIm5iZiI6MTcwMjM1NDQ1MSwicGF0aCI6Ii81OTY2ODY0Ny8yODk2NDQyMDgtMzI3MmU2NjItNWRiNS00NzYyLWFiMjgtZjQ3N2M4OWUxY2YzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjEyVDA0MTQxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWViMWZkNDIwM2FmMDlmNjJjOTExY2Q5ZjU3Yzg2MzNjMWRhMGY3OWUzZWE2M2YyOTcwMjljYzMxNjg5YTBhYTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.z4feCavvj3bmE47ipEHll1ey9tglPto7ZBm_DX2Bnqo">

@lijenn - Now, it should work. Merge was pending on GitHub. Sorry for inconvenience.
![image](https://github.com/lzim/teampsd/assets/144898781/5b15f6d7-81c0-44ca-9c1a-01e033dac005)

**Comment by lijenn** [comment]
Hi @tejaspesquare Any updates on 15, 16, and 26?

**Comment by tejaspesquare** [comment]
> Hi @tejaspesquare Any updates on 15, 16, and 26?

@lijenn - Please see my response below 
Point 15 - Team will be focused on Casual Loop Diagrams till 18th or 19th. After this they will start looking into Vault API stuff which David provided on our query for filter worlds.
Point 16 - Its working on our end. Can you please check again if its working or not.
Point 26 - This will be available to you by 19th or 20th Dec.

Thanks!

**Comment by ljmoody** [comment]
Hi @tejaspesquare. @lijenn and I had a Sim UI test session today. We're following up with the following:

Is there an update for Point 15?
>  15. Hide the Shared world for all users that do not have the administrative_login privilege. Only users who have the administrative_login should be able to see all the registered teams, ind, and shared worlds.

Is there an update for Point #26?
>  26. When the screen is loading after clicking the green "GO!" button, include quick tips while the page is loading:

**New request:**
- [ ] 27. Please increase the font size of the highlighted text to be the same size as "Looks like you are the first person..." above:
![image](https://github.com/lzim/teampsd/assets/124643236/82578fd0-3f5f-45b3-bb86-b7a49bef6a5d)

**Comment by tejaspesquare** [comment]
@ljmoody - 
1. Point 26 - Quick tips available on TEST-Slow. Can you please check once at your end too. 

![image](https://github.com/lzim/teampsd/assets/144898781/665d3e5b-84bc-4a10-af94-0ad7ca9df642)

2. Point 27 - This is fixed & will be moved to TEST-Slow by EOD.
3. Point 15 & 17 - We have just begun to look into this requirement. We need to explore Vault API underlying functionalities & its implementation for our problem. Expecting series of communications with David (Epicenter) for this matter. Will keep you informed of any developments.

**Comment by lijenn** [comment]
Hi @tejaspesquare, for 26. quick tips, I do not see it in Test-Slow yet:
<img width="1198" alt="image" src="https://github.com/lzim/teampsd/assets/59668647/932a1a86-bd5f-4f27-ab96-ce8e4ff9113d">

**Comment by tejaspesquare** [comment]
Hi @lijenn : Point 26 - Thank you for notify.  This is resolved now & moved to Test-Slow.

**Comment by ljmoody** [comment]
Hi @tejaspesquare and thank you for your updates. I verified that: 

- Point 26, adding quick tips, is now functional:
![image](https://github.com/lzim/teampsd/assets/124643236/3db044a0-34a7-450c-a5c6-80a513efa614)

- Point 27, font is enlarged:
![image](https://github.com/lzim/teampsd/assets/124643236/bb41de58-3457-44f6-8c6e-68ffef972d8e)

FYI @lijenn

**Comment by ljmoody** [comment]
Hi, @tejaspesquare  James let us know you're making progress on the API issue. Do you have any further updates for us?

**Comment by tejaspesquare** [comment]
@ljmoody - We are coordinating with Daivd. So far, we have successfully inserted sample data through vault API. Now, we will look into how actual objects need to bind & save using Vault API.

**Comment by tejaspesquare** [comment]
@ljmoody , @lijenn - Point 15. Hide the Shared world for all users that do not have the administrative_login privilege. -> This is moved to TEST-Slow. Kindly test & provide your feedback.
Note: At moment have verified with self-registered accounts.

**Comment by ljmoody** [comment]
Jenn and Laura have completed testing, so we are closing this card. Point 17 remains, and is being tracked as a bug under #3091.

**Comment by hnorthcraft** [comment]
@lijenn doe point 15 mean the same thing as 15SP?

**Comment by lijenn** [comment]
@hnorthcraft No, Laura is referencing point 15 to a checklist item in this test card.

**Comment by hnorthcraft** [comment]
@lijenn ahh okay. can you let me know how many SP for this card then? lots of work here :)

**Comment by lijenn** [comment]
@hnorthcraft #adjudicate_memo

13 SP since it looks like we did full cadence amounts of work to test, implement dev changes, and tested/trained with users on this feature.
